pkgs/applications/video/mkvtoolnix/default.nix

```diff
@@ -25,7 +25,7 @@ stdenv.mkDerivation rec {
     expat
     file xdg_utils boost libebml zlib
     libmatroska libogg libvorbis flac
-    (optional withGUI qt5.qtbase)
+    (optional withGUI qt5.qtbase.dev)
   ];
```

Comment:
@rnhmjoj It means that `optional withGUI qt5.qtbase` (when `withGUI` is true) is a list with one element. This one element is itself a list with many elements. This situation confuses `mkDerivation` and because of that only the head of the inner list is taken into account. Right?